compendium/test_bundles/TestInterfaces/include/TestInterfaces/Interfaces.hpp

```diff
@@ -112,6 +126,18 @@ namespace test
       virtual bool IsDeactivated() = 0;
       virtual ~LifeCycleValidation();
     };
+
+    class US_TestInterfaces_EXPORT TestManagedServiceFactory {
+      public:
+        TestManagedServiceFactory() = default;
+        virtual ~TestManagedServiceFactory() = default;
+
+        virtual int getUpdatedCounter(std::string const& pid) = 0;
+
+        virtual int getRemovedCounter(std::string const& pid) = 0;
+
+        virtual std::shared_ptr<TestManagedServiceFactoryServiceInterface> create(std::string const& config) = 0;
+    };
```

Comment:
I think it'd be nice to have the "TestManaged..." interfaces all close to each other by moving this interface up to where the other two added interfaces are (around line 65)